Formula/toilet.rb

```diff
@@ -15,10 +16,13 @@ class Toilet < Formula
     sha256 "ef2c34f742b366f84d2aeeb6d83fb94d6bd443f210e56968fb8b2b5700eab759" => :mavericks
   end
 
+  depends_on "autoconf" => :build if build.head?
+  depends_on "automake" => :build if build.head?
```

Comment:
```suggestion
  if build.head?
    depends_on "autoconf" => :build
    depends_on "automake" => :build
  end
```

Reply:
Or better yet, put it in a `head do`:

```suggestion
  head do
    url "https://github.com/cacalabs/toilet.git"
    
    depends_on "autoconf" => :build
    depends_on "automake" => :build
  end
  
```

and remove the `head` line further up.

Reply:
Nice!

Reply:
Implemented the latter!  Thanks @Bo98 and @issyl0!